Bio/SeqUtils/NucUtils.py

```diff
@@ -0,0 +1,191 @@
+# Copyright 2018 Aleksandra Jarmolinska.  All rights reserved.
+# This file is part of the Biopython distribution and governed by your
+# choice of the "Biopython License Agreement" or the "BSD 3-Clause License".
+# Please see the LICENSE file that should have been included as part of this
+# package.
+"""Functions for dealing with GC/AT content."""
+
+
+from __future__ import print_function
+
+__GC_CONTENT_VALUE = {
```

Comment:
Why the double leading underscore, isn't the convention one leading underscore for a private variable?

Reply:
I believe PEP8 allows double leading underscore for "very private" variables, and there is some name mangling involved for use in classes which are supposed to be subclassed. Although you are - here the "normal" one underscore is absolutely sufficient